gui-agent/qubes-gui-runuser.c

```diff
@@ -320,5 +324,18 @@ int main(int argc, char **argv) {
     signal(SIGTERM, propagate_signal);
     signal(SIGHUP, propagate_signal);
 
-    return do_execute(argv[1], argv[2], argv+3);
+    char *user = argv[1];
+    unsigned int len = 0;
+    if (user[0] == 0) {
```

Comment:
This behavior needs to be documented somewhere. At the very least in the `usage()`.